appender/influxdb/src/main/java/org/apache/karaf/decanter/appender/influxdb/InfluxDbAppender.java

```diff
@@ -95,10 +96,10 @@ public void activate(Dictionary<String, Object> config) {
         String prefix = "tag.";
         for (Enumeration<String> e = config.keys(); e.hasMoreElements(); ) {
             String key = e.nextElement();
-            if (key.startsWith(prefix)) {
+            if( key.startsWith(prefix)) {
                 Object value = this.config.get(key);
                 if (value != null)
-                    tags.put(key.substring(4), value.toString());
+                    globalTags.put(key.substring(4),value.toString());
```

Comment:
I think a better name would be `tagsNames`.

Reply:
I wanted the difference between event tags and global tags to be explicit.
global tags will be sent with every metric.( for example application name).
event tags are specific for a single event.